docs/guides/benchmarks.md

```diff
@@ -0,0 +1,162 @@
+# Benchmarks
+
+You benchmark your machine learning code in nnbench by encapsulating the benchmarking code in separate functions and applying one of the decorators provided. 
+The available decorators are 
+- `@benchmark`, which runs a benchmark with supplied parameters,
+- `@parametrize`, which runs several benchmarks with the supplied parameter configurations,
+- `@product`, which runs benchmarks with all parameter combinations that arise from the supplied values. 
```

Comment:
We might want to give all of them as absolute imports like `@nnbench.benchmark`, since that's what we have done so far in tests and examples.